index.html

```diff
@@ -288,6 +290,53 @@ <h5>LET'S CONNECT</h5>
 
     </section>
 
+    <section class="contact_section">
+
+        <div class="contact_form">
+
+            <header>
+                <h3>Contact me</h3>
+            </header>
+
+            <p>
+                If you have an application you are interested in developing, a feature that you need built or a project
+                that
+                needs coding. I’d love to help with it
+            </p>
+
+            <form   action="https://formspree.io/f/xdojpwrd" method="POST">
+
+                <fieldset class="contact_info">
+
+                    <label>
+                        <input class="input1" maxlength="30" type="text" placeholder="Yeremias" id="" value="" name="" required>
+                    </label>
+
+                    <label>
+                        <input class="input1"  type="email" placeholder="natajayanj14@gmail.com" id="" value="" name="" required>
+                    </label>
+
+                    <label>
+                        <textarea class="textarea1" placeholder="Write your message here" name="" id="" maxlength="500" rows="10" required></textarea>
+                    </label>
```

Comment:
- [ ] Great job implementing Formspree service! 💯 However, you need to set the name attribute for each input in order for it to work. Please fix this! 🛠️ 

![image](https://user-images.githubusercontent.com/77069011/199484534-27f70e61-c742-4165-ae98-0ec9bd8a5218.png)